src/main.js

```diff
@@ -1,17 +1,47 @@
 import IndexPageModes from './layouts/IndexPageModes/IndexPageModes';
 import ButtonsContainer from './layouts/buttonsContainer/buttonsContainer';
+import StarWarsIntro from './layouts/StarWarsIntro/StarWarsIntro';
+import { hideStarWarsIntro } from './utils/hideStarWarsIntro';
+import { tapHandler } from './utils/doubleTapHandler';
 import changeGameMode from './utils/changeGameMode';
 
 const indexPage = new IndexPageModes();
+const starWarsIntro = new StarWarsIntro();
 
 const buttonsContainer = new ButtonsContainer();
 
+document.body.appendChild(starWarsIntro.render());
+
 document.getElementById('main-grid-container').appendChild(indexPage.render()).appendChild(buttonsContainer.render());
 
+setTimeout(function () {
```

Comment:
Correct me if I'm wrong but it seems the timeout could be shorter. In the future you could consider implementing it the way you originally intended to make the solution is better suited for changes of the length of the text on narrower and wider screens but this should be enough for now.